source/extensions/filters/http/gcp_authn/gcp_authn_filter.h

```diff
@@ -65,8 +65,8 @@ class GcpAuthnFilter : public Http::PassThroughFilter,
 
 private:
   GcpAuthnFilterStats generateStats(const std::string& stats_prefix, Stats::Scope& scope) {
-    // const std::string prefix = absl::StrCat(stats_prefix, "gcp_authn.");
-    return {ALL_GCP_AUTHN_FILTER_STATS(POOL_COUNTER_PREFIX(scope, stats_prefix))};
+    const std::string prefix = absl::StrCat(stats_prefix, "gcp_authn.");
```

Comment:
Isn't this a functional change? Do docs, etc. need to be updated here?

/wait

Reply:
It is a minor change more about the naming of the stats. But I do agree the doc for `statistics` is needed. 
The implementation of this filter extension is not fully done yet. There is another PR for cache which will be sent out soon. I am planning to complete the doc (right now is just basic version) along with final piece of this filter. 
I added a TODO here as the reminder.

Reply:
Mark as resolved since stats naming change is removed from this PR. It will be added in a separate PR along with docs change.
Thanks for pointing it out!